app/assets/stylesheets/active_admin/components/_dropdown_menu.scss

```diff
@@ -1,151 +1,45 @@
-.dropdown_menu {
-  display: inline;
-
-  .dropdown_menu_button {
-    @include light-button;
-    position: relative;
-    padding-right: 22px !important;
-    cursor: pointer;
-
-    &:before {
-      content: ' ';
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-width: 3px 3px 0;
-      border-style: solid;
-      border-color: #FFF transparent;
-      right: 12px;
-      top: 45%;
-    }
-
-    &:after {
-      content: ' ';
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-width: 3px 3px 0;
-      border-style: solid;
-      border-color: #777 transparent;
-      right: 12px;
-      top: 45%;
-    }
-  }
-
-  .dropdown_menu_nipple {
-
-    // The nipple's border
-    content: "";
-    position: absolute;
-    top: -6px;
-    display: block;
-    width: 0;
-    height: 0;
-    border-width: 0 6px 6px;
-    border-style: solid;
-    border-color: darken($primary-color, 4%) transparent;
-    z-index: 100;
-
-    // The nipple's inner shadow
-
-    &:before {
-      content: ' ';
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-width: 0 5px 5px;
-      border-style: solid;
-      border-color: lighten($primary-color, 15%) transparent;
-      left: -5px;
-      top: 1px;
-    }
-
-    // The nipple's background color
-
-    &:after {
-      content: ' ';
-      position: absolute;
-      width: 0;
-      height: 0;
-      border-width: 0 5px 5px;
-      border-style: solid;
-      border-color: lighten($primary-color, 4%) transparent;
-      left: -5px;
-      top: 2px;
-    }
-  }
+.caret {
+  border-left: $caret-width-base solid transparent;
+  border-right: $caret-width-base solid transparent;
+  border-top: $caret-width-base dashed;
+  display: inline-block;
+  height: 0;
+  margin-left: 2px;
+  vertical-align: middle;
+  width: 0;
+}
 
-  .dropdown_menu_list_wrapper {
-    display: inline-block;
-    position: absolute;
-    background-color: white;
-    padding: 2px;
-    box-shadow: rgba(0,0,0,0.4) 0 1px 3px, lighten($primary-color, 15%) 0px 1px 0px 0px inset;
-    background-color: $primary-color;
-    @include gradient(lighten($primary-color, 4%), darken($primary-color, 5%));
-    border: solid 1px darken($primary-color, 10%);
-    border-top-color: darken($primary-color, 4%);
-    border-bottom-color: darken($primary-color, 17%);
-    border-radius: 4px;
-    z-index: 2000;
-    display: none;
+.dropdown {
+  display: inline-block;
+  position: relative;
+}
 
-    .dropdown_menu_list {
+.dropdown-menu {
+  background-color: $dropdown-background-color;
+  border: 1px solid $dropdown-border-color;
+  border-radius: $border-radius-base;
+  box-shadow: 0 6px 12px rgba(0, 0, 0, .175);
+  display: none;
+  left: 0;
+  list-style: none;
+  margin: 2px 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `2px 0 0` instead of `2px 0 0 0`